docs/zuri_app(desktop).md

```diff
@@ -0,0 +1,239 @@
+# Zuri Chat App (Desktop)
+## `Overview`
+ZuriChat is a collaboration hub that brings the right people and tools together to get work done. From Fortune 100 companies to corner markets, millions of people around the world use ZuriChat to connect their teams, unify their systems, and drive their business forward.
+
+## `Goals`
+- Provide a platform for team colaboration
```

Comment:
Check the spelling of collaboration

Reply:
1. Remove ` ` from the headings
2. The goals should be the company goals, reach out to the vision and mission team for the right company goals.
3. Double-check your spelling and sentence structure with Grammarly. e.g their not thier.

Reply:
what should I remove?

Reply:
done